R/sim_data_list.R

```diff
@@ -267,3 +267,18 @@ setMethod(
     )
   }
 )
+
+# get_sim_data ----
+#' @rdname get_sim_data
+setMethod("get_sim_data", "SimDataList", function(sim_data_list, index = NULL, dataset = NULL) {
+  
```

Comment:
Maybe some assertions here for index and dataset being integerish if not null
```suggestion
  assert_integerish(index, lower = 1, upper = length(sim_data_list@data_list))
  assert_integerish(dataset, lower = 1)
```